### DISCUSSION TITLE
Request: Simple Essentials Economy Tax System

### DISCUSSION THREAD

**Discussion by SteeltonMC** [EssentialsX/Essentials#4372] [Ideas and Feature Suggestions] [answered]
Hello, this may seem like an unimportant suggestion because Spigot does not have any updated plugin since 1.17 yet, and there was only one single plugin that is somewhat updated.

Basically, every day or week, a configurable amount is taken from a user's balance. Perhaps it may also include Tax Brackets depending on the balance. I would really love this feature! 

Example Config?

                Poor:
                    MaxBalance: 500.00
                    TaxType: PERCENT
                    TaxAmount: 1.0
                Rich:
                    MaxBalance: 50000.00
                    TaxType: PERCENT
                    TaxAmount: 10.0

Please let me know what you think!

**Comment by Chew** [comment]
I think this is #1480?

**Comment by SteeltonMC** [comment]
That seems to revolve around player to player /pay command.

This would be on a schedule effecting players economy balance. I would love
to see my suggestion get considered if possible. :)

**Comment by JRoy** [comment, marked as answer]
This is out of the scope of Essentials, should be either done by a separate plugin using Vault.
